### PR TITLE
Add copy to app

### PR DESCRIPTION
**Pull request by fungjj92**
## Overview

The client provided copy to update around the app, which is full of lorem-ipsum placeholder text. This PR captures all requested changes to date with provided copy.

Connects #433 

### Demo

The only really new thing are tooltips. They're pretty basic.

screen shot 2019-01-31 at 4 38 37 pm

### Notes

Still awaiting some text from client.

## Testing Instructions

Most of the copy was copy & paste around the app.

Some of the changes are in email copy -- go through a sign up, activate your account, and resend activation link to make sure you see those changes in your console running `./scripts/debugserver.sh`. The check is not ideal but works I think -- the django templates only receive so much relevant info (I couldn't get a full path name to check for the `?beekeepers` querystring for example. If I missed something, lemme know).

### CODE REVIEW

**Comment by rajadain** [comment]
Taking a look now.

**Comment by rajadain** [comment]
The activation email text for me was still the old one:

```
You (or someone pretending to be you) have asked to register an account at
localhost:8000.  If this wasn't you, please ignore this email
and your address will be removed from our records.


To activate this account, please click the following link within the next
7 days:


http://localhost:8000/accounts/activate/6ad7a0f67b2e74c489d29b779587a1396d18f97f/




    Sincerely,
    localhost:8000 Management
```

and the URL still said "Bee Pollinator" when testing locally.

**Comment by fungjj92** [comment]
Can you confirm that the user you were using locally had `userprofile.origin_app` as Beekeepers?

**Comment by rajadain** [comment]
Yes

```sql
SELECT username, origin_app
FROM auth_user a INNER JOIN user_userprofile p ON a.id = p.user_id;

 username | origin_app
----------+------------
 test1    | Beekeepers
 test2    | Beekeepers
 test3    | Beekeepers
(3 rows)
```

**Comment by rajadain** [comment]
Out of efficiency, I added a commit to slightly improve the styling of the tooltips, which were really dense and hard to parse:

image

**Comment by rajadain** [comment]
I wonder if it's because `RegistrationView` is instantiated before the `UserProfile` object is created:

https://github.com/project-icp/bee-pollinator-app/blob/dcaeca1eed1b6395da6d019868c141c970612df0/src/icp/apps/user/views.py#L138-L150

I added `user.userprofile.origin_app` to the activation email template and it prints as an empty string.

Also, we need to update that to include the case of `app.beescape.org` being in the request host.

**Comment by rajadain** [comment]
Let's add this as a commit, which was missed in f48ea080.

```diff
diff --git a/src/icp/apps/user/views.py b/src/icp/apps/user/views.py
index baf4cd71..6517d276 100644
--- a/src/icp/apps/user/views.py
+++ b/src/icp/apps/user/views.py
@@ -139,9 +139,11 @@ def sign_up(request):
     form = RegistrationFormUniqueEmail(request.POST)
 
     if form.is_valid():
+        beekeepers = ('app.beescape.org' in request.get_host() or
+                      'beekeepers' in request.GET)
         user = view.register(request, form)
         origin_app = \
-            UserProfile.BEEKEEPERS if 'beekeepers' in request.GET else \
+            UserProfile.BEEKEEPERS if beekeepers else \
             UserProfile.POLLINATION
 
         UserProfile.objects.create(
```

**Comment by fungjj92** [comment]
I'm also seeing the blank string. In fact, in some cases `user` is anonymous, and `site` isn't even passed though. In particular I'm going through the forgot PW workflow. Hm. We need more reliable template hooks, then.

**Comment by fungjj92** [comment]
That said if the domain isn't even being sent to the template, I feel like we're out of luck and have to have static universal text.

**Comment by fungjj92** [comment]
Only some Django templates are customizable based on limited data fed to the templates (out of our control). 

Before:
screen shot 2019-02-04 at 1 15 38 pm

After:
screen shot 2019-02-04 at 1 37 52 pm


Sign up email:
screen shot 2019-02-04 at 1 53 03 pm


Resend activation link email:
screen shot 2019-02-04 at 1 38 13 pm


Forgot PW:
screen shot 2019-02-04 at 1 58 43 pm

**Comment by rajadain** [comment]
Hmm the commit I added seems to have been lost. I'll add it again.

**Comment by fungjj92** [comment]
Oops, my bad. Forgot to pull down your commit.

**Comment by fungjj92** [comment]
Thanks for the good hustle 🦅
